src/libs/test.ts

```diff
@@ -0,0 +1,37 @@
+counter = 0;
```

Comment:
@trag-bot Global variable 'counter' may lead to unexpected behavior when accessed concurrently in a multi-threaded environment.